package.json

```diff
@@ -98,7 +98,8 @@
           "type": "string",
           "enum": [
             "off",
-            "messages"
+            "messages",
+            "verbose"
```

Comment:
Have you tested this change? Is this setting documented somewhere? (I dont know about it, so this is for my own education)